lms/djangoapps/course_api/blocks/serializers.py

```diff
@@ -70,7 +77,6 @@ class BlockDictSerializer(serializers.Serializer):  # pylint: disable=abstract-m
     Serializer that formats a BlockStructure object to a dictionary, rather
     than a list, of blocks
     """
-    root = serializers.CharField(source='root_block_usage_key')
```

Comment:
@lenacom I now saw your comment to @nedbat below.  I'll have @nasthagiri review to make sure this is as she expected.